zcash_transparent/src/bundle.rs

```diff
@@ -230,6 +230,14 @@ pub struct TxOut {
 }
 
 impl TxOut {
+    // Constructs a new `TxOut` from its constituent parts.
+    pub fn new(value: NonNegativeAmount, script_pubkey: Script) -> Self {
```

Comment:
Document this addition in the changelog. (Also it is currently unnecessary as the struct is fully public, but I agree with transitioning these types to constructors over time.)